bootstrap-obvious-ci-and-miniconda.py

```diff
@@ -1,103 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Not trying to be obtuse. Only pointing out it is still available and that we had tried to protect against problems in conda-forge. In any event, PR ( https://github.com/conda-forge/conda-smithy/pull/376 ) will add it back. Feel free to merge.